.github/workflows/coverity.yaml

```diff
@@ -0,0 +1,48 @@
+name: Coverity
+on:
+#  schedule:
+#    # run daily 20:00 on master branch
+#    - cron: '0 20 * * *'
+  push:
+    branches:
+      - coverity_scan
+jobs:
+  coverity:
+    name: Coverity
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        pg: [12.3]
```

Comment:
We might have to run this on PG11 as well since since we have different code paths for the two versions.